autoload/lsp/diag.vim

```diff
@@ -410,12 +410,17 @@ enddef
 # Display the diagnostic messages from the LSP server for the current buffer
 # in a location list
 export def ShowAllDiags(): void
+  var save_winid = win_getid()
```

Comment:
should move to/after `if !DiagsUpdateLocList(bufnr())`

Reply:
Done 👍